app/services/saml_request_validator.rb

```diff
@@ -43,7 +43,7 @@ def parsed_vectors_of_trust
     return @parsed_vectors_of_trust if defined?(@parsed_vectors_of_trust)
 
     @parsed_vectors_of_trust = begin
-      if vtr.is_a?(Array) && !vtr.empty?
+      if vtr.blank?
```

Comment:
I think we need `.present?` because if `vtr` is blank, then the map on the next line will have nothing?

Reply:
Yep. At least we know the tests work.